prqlc/prqlc/src/debug/render_html.rs

```diff
@@ -64,40 +66,35 @@ fn write_debug_log<W: Write>(w: &mut W, debug_log: &DebugLog) -> Result {
                     .unwrap_or_default();
                 let stage = stage.as_ref().to_lowercase();
 
-                writeln!(w, "<br/><span class=muted>{stage} {substage}</span><hr/>")?;
+                writeln!(w, r#"<h2 class="muted">{stage} {substage}</h2>"#)?;
             }
             DebugEntryKind::Message(message) => {
                 write_message(w, message)?;
             }
             _ => {
-                write_titled_entry(w, entry, index)?;
+                write_titled_entry(w, entry)?;
             }
         }
 
         writeln!(w, "</div>")?; // entry
     }
-    writeln!(w, "</div>")?; // entries
 
+    writeln!(w, "</main>")?;
     writeln!(w, "</body>")?;
+    writeln!(w, "</html>")?;
 
     Ok(())
 }
 
-fn write_titled_entry<W: Write>(w: &mut W, entry: &DebugEntry, index: usize) -> Result {
-    let entry_id = format!("entry-{index}");
-
-    writeln!(
-        w,
-        "<input id={entry_id} class=entry-collapse type=checkbox>"
-    )?;
-
-    writeln!(w, r#"<label for={entry_id} class="entry-label clickable">"#)?;
+fn write_titled_entry<W: Write>(w: &mut W, entry: &DebugEntry) -> Result {
+    writeln!(w, "<details open>")?;
+    writeln!(w, r#"<summary class="entry-label clickable">"#)?;
```

Comment:
Oh yeah, I forgot about details/summary. Nice.

Reply:
This change loses collapsed status between page refresh. Can we revert?

Reply:
Did it preserve the collapse status between page refresh before? How did it do that?

Is it even desirable? When the user refreshes the page in the browser the HTML file might have been regenerated and section might not exist. I suppose the open/close state could preserved using the [sessionStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) API. [[[Category:Computer companies established in 1979]]](https://alexwlchan.net/2020/remembering-if-a-details-element-was-opened/)

Reply:
I agree we should try and retain the collapsed status, would favor reverting first and then figuring it out...